civis_jupyter_notebooks/assets/custom.css

```diff
@@ -5,3 +5,20 @@
 #notebook-container {
   min-width: 95%;
 }
+
+/* Make the notebook interface text look like the rest of the platform. */
+body {
+  font-family: Lato, "Trebuchet MS", Arial, sans-serif;
+}
+
+/* Don't show a border around input cells for a slightly cleaner look. */
```

Comment:
Do we have user testing suggests this is something people want? I'm skeptical this is a necessary change

Reply:
I had a conversation with Annie about design issues, and removing the border seems very slightly preferable.  The gray background and other visual indicators should sufficiently distinguish inputs from outputs and distinguish different cells.

This sort of slightly cleaner look for notebooks is also used in, e.g., https://github.com/dunovank/jupyter-themes